components/admin_console/admin_console.jsx

```diff
@@ -149,6 +157,7 @@ export default class AdminConsole extends React.Component {
                 <div className='admin-console'>
                     <AdminSidebar/>
                     <Switch>
+
```

Comment:
Was this accidentally change?

Reply:
Good catch -- yes, it was a not-completely-removed-earlier-attempt-at-fixing-this. Fixed :)